functions.php

```diff
@@ -898,6 +898,8 @@ function twentytwenty_get_customizer_css( $type = 'front-end' ) {
 		// Get variables
 		$accent 		= get_theme_mod( 'twentytwenty_accent_color' );
 		$accent_default = '#CD2653';
+		$text 		= get_theme_mod( 'twentytwenty_text_color' );
+		$text_default = '#1A1B1F';
```

Comment:
```suggestion
		$text_default = '#1a1b1f';
```

The WordPress CSS Coding Standards state that color values must all be lower-case:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#properties